scripts/test_manager/configurator.py

```diff
@@ -86,6 +86,8 @@ def _pre_processing_check(self, test):
         """Check if test is valid to start processing."""
         if not self._process_known_failure and test.metadata.known_failure:
             raise KnownFailureError("")
+        if test.metadata.is_manual:
```

Comment:
I don't think we want to block it's execution. The `manual` doesn't mean it cannot be started, only that you have to put in something custom which is not available for automatic run. In this case it's custom boot.iso which is not available on Fedora.

Reply:
Good point, sorry. I will fix that.